futurenhs.app/components/GroupPageHeader/index.tsx

```diff
@@ -219,6 +222,17 @@ export const GroupPageHeader: (props: Props) => JSX.Element = ({
                             desktop={3}
                             className={generatedClasses.actionsWrapper}
                         >
+                            {shouldRenderPendingMessage &&
+                                !shouldRenderGroupJoinLink && (
+                                    <div className="nhsuk-body-m">
+                                        <SVGIcon
+                                            name="icon-clock"
+                                            className="u-w-4 u-h-4 u-mr-2"
+                                        />
+                                        Awaiting Approval
```

Comment:
Pass in via content config if possible

Reply:
Can't in this one unfortunately as it's not a page template so doesn't use the text service. Rest of the strings in this component are hardcoded - actions, join group etc.